city_scrapers/spiders/chi_ssa_20.py

```diff
@@ -0,0 +1,105 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa20Spider(CityScrapersSpider):
+    name = "chi_ssa_20"
+    agency = "Chicago Special Service Area #20 South Western Avenue"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+    location = {
+        "name": "Beverly Bank & Trust,",
+        "address": "10258 s. Western ave.",
+    }
+
+    def parse(self, response):
+
+        base = response.xpath(
+            "//*[self::p or self::strong or self::h3]/text()"
+        ).getall()
+
+        base = [re.sub(r"\s+", " ", item).lower() for item in base]
+
+        for index, line in enumerate(base):
+            if "ssa meetings" in line:
+                del base[:index]
+
+        for index, line in enumerate(base):
+            if "ssa 64" in line:
+                del base[index:]
+
+        for item in base:
+            if re.match(r"^\D*\d{4}\D*$", item):
+                year = re.match(r"^\d{4}", item)[0]
+
+        for item in base:
+
+            # don't pass empty lines to methods
+            if re.match(r"^\s*$", item):
+                continue
+
+            start = self._parse_start(item, year)
+            if not start:
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=start,
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "SSA 20"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item, year):
+
+        if not any(word in item for word in ["beverly", "ssa"]):
+            item = re.sub(r"([,\.])", "", item).strip()
+            ready_date = item + " " + year
+            date_object = datetime.strptime(ready_date, "%A %B %d %I %p %Y")
```

Comment:
We've run into fewer errors ignoring the day of the week `%A` because it doesn't add info and can sometimes have typos.

Reply:
Fixed.